Makefile

```diff
@@ -128,7 +128,7 @@ deploy: build
 
 .PHONY: buildprod
 buildprod: docker-compose.yml
-	docker compose run --rm client yarn build
+	docker compose run --rm client yarn type-check-build
 	docker compose run --rm server yarn build
```

Comment:
is there a reason to not add type checking to the client's default `yarn build` (i.e., replace it instead of adding another command)

Reply:
Not really, no. I did it this way so it doesn't actually run on a real deploy but that doesn't a ton of sense since we'd want to fix anything before deploying. I'll switch them around